lib/addons/src/make-decorator.test.ts

```diff
@@ -1,10 +1,10 @@
-import { StoryContext, StoryGetter } from './types';
+import { StoryContext } from './types';
 import { makeDecorator } from './make-decorator';
 
 // Copy & paste from internal api: client-api/src/client_api
-type DecoratorFn = (fn: StoryGetter, context: StoryContext) => any;
+type DecoratorFn = (fn: any, context: StoryContext) => any;
```

Comment:
Any particular reason for making this `any` and not keeping `StoryGetter`?

Reply:
StoryGetter doesn't exist it's already any, it must have been removed at some point or something

Reply:
Yikes, seems like it was removed back in [August](https://github.com/storybookjs/storybook/commit/40641489624b5ee2c2532a918bfa0809011b02ab)!